### DISCUSSION TITLE
Orbstack support?

### DISCUSSION THREAD

**Discussion by alexandrosk** [wardenenv/warden#659] [Ideas]
Today I learned about [orbstack](https://orbstack.dev/) and public beta

Im trying to set it up with warden, but let me know if anyone did this before.

Edit:
running `warden svc up` seems to work but I Get an error after traefik on `warden env up`

```
parsing /Users/username/.warden/environments/magento2/magento2.base.yml: yaml: unmarshal errors:
  line 34: mapping key "<<" already defined at line 33
  line 35: mapping key "<<" already defined at line 33
```

Thanks!

**Comment by bap14** [comment]
@alexandrosk  This doesn't seem like a core warden issue. The `<<` sequence doesn't appear as an actual key in any yaml file in the project (see: https://github.com/search?q=org%3Awardenenv+%3C%3C+path%3A*.yml&type=code ). Warden doesn't by default use the merge key syntax, it uses anchors and aliases. Have you edited the core environment files of your local warden installation?

**Comment by alexandrosk** [comment]
Wow thanks, so strange that I didn't have the default one, reinstalling seems like its working fine!
